amazonorders/lib/transactions.py

```diff
@@ -0,0 +1,62 @@
+__copyright__ = "Copyright (c) 2024 Jeff Sawatzky"
```

Comment:
Once you move the function for getting transaction history to its own `transactions.py` file, let's move this in there too (prefixed with `_`), so we don't have to have this `lib` module.